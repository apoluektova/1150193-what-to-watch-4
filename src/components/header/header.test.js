import React from "react";
import renderer from "react-test-renderer";
import Header from "./header.jsx";

const userInfo = {
  id: 1,
  email: `ben@mail.ru`,
  name: `Ben`,
  avatarUrl: `/wtw/static/avatar/5.jpg`,
};

it(`ErrorMessage should render correctly`, () => {
  const tree = renderer
     .create(<Header
       authorizationStatus={`AUTH`}
       authInfo={userInfo}
       onSignInClick={() => {}}
     />)
     .toJSON();

  expect(tree).toMatchSnapshot();
});
