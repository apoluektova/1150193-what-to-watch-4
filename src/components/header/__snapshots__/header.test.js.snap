// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage should render correctly 1`] = `
<header
  className="page-header movie-card__head"
>
  <div
    className="logo"
  >
    <a
      className="logo__link"
    >
      <span
        className="logo__letter logo__letter--1"
      >
        W
      </span>
      <span
        className="logo__letter logo__letter--2"
      >
        T
      </span>
      <span
        className="logo__letter logo__letter--3"
      >
        W
      </span>
    </a>
  </div>
  <div
    className="user-block"
  >
    <div
      className="user-block__avatar"
    >
      <img
        alt="Ben avatar"
        height="63"
        src="/wtw/static/avatar/5.jpg"
        width="63"
      />
    </div>
  </div>
</header>
`;
